### ISSUE TITLE
Handle duplicate muscles on muscle overview

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#145]
Currently `MuscleWidget` simply renders the list of main and secondary muscles given to it. When a muscle is both main and secondary, only the main background should be drawn (this is something that, if the exercises are saved sensibly should not happen, but we still need to account for it)

![Screenshot_1648498312](https://user-images.githubusercontent.com/2280729/160479745-970e3c2d-bb9a-4c20-8a38-900df5e53701.png)

**Comment by makwasi1** [comment]
Hello, @rolandgeider is this issue still open. If so could you provide more information I would love to try it out?

**Comment by rolandgeider** [comment]
Hi @makwasi1 sure go ahead! This is available in the `feature/exercise-crowdsourcing` branch, if you go to the workout tab you can access the add exercise form and select the different muscles for the new exercise (note that at the moment there's still a bug so that you need to click on the dropdown to unselect the muscles). Do you already have the app installed? You will need a backend for this, you can write me an email and I can give you access to the test server

**Comment by sarkartanmay393** [comment]
@rolandgeider I having an issue with setting up the development environment of this flutter project. Can you guide me towards it ? And I need a little more explanation of this issue.

**Comment by rolandgeider** [comment]
hi @sarkartanmay393 ! I'm guessing you are having problems about the server you need to use? I have sent you an email

As for the issue: we have a widget that generates the muscle overview from the screenshot. For example, when adding a new exercise it is possible to select which primary and secondary muscles the new exercise trains it shows which entries were selected. If a certain muscle was selected both as primary and secondary, the secondary one should not be drawn

**Comment by sandimilohanic** [comment]
hi @rolandgeider , i don't find a screen for Add exercises, is this screen correct?

![Screenshot_69](https://user-images.githubusercontent.com/7765448/198866223-1fbd8bb2-aa77-4fc9-9ddc-b7df3abc1f36.png)

**Comment by rolandgeider** [comment]
Hi @sandimilohanic , yes that's the screen

**Comment by kkankariya007** [comment]
Hello, @rolandgeider can you assign me this issue, ill like to work on it?

**Comment by rolandgeider** [comment]
Hi @kkankariya007 , go ahead

**Comment by rolandgeider** [comment]
(BTW, this was merged in master some time ago)

**Comment by kkankariya007** [comment]
Like has it been solved ?

**Comment by rolandgeider** [comment]
No, no, I just saw that in one of the comments I wrote that the exercise contribution (where this widget was first implemented) was being developed in another branch and that one was merged

**Comment by ASP124** [comment]
Hello @rolandgeider , I recently viewed your project and it is really very well fabricated project though there are minor issues which can be resolved with ease. So I request you to grant me to work on this issue. 
Regards ASP.

**Comment by tejanshsachdeva** [comment]
Is this issue still open to be resolved or can it be assigned to me?

**Comment by jagrutatwork** [comment]
Is anyone working on this issue? I would like to accompany you,  I am a beginner and really curious to learn more.

**Comment by renusuda** [comment]
@rolandgeider 
It looks like this issue has already been resolved by the code introduced in [this PR](https://github.com/wger-project/flutter/pull/422).
If that's the case, would it be okay to close it?

Thanks for your work on this project!
